bodybuilder/bodybuilder.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for bodybuider v2.2.0
+// Project: https://github.com/danpaz/bodybuilder
+// Definitions by: Ismael Slimane Zubillaga <https://github.com/islimane>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "bodybuilder" {
```

Comment:
Rename this file to `index.d.ts`, and ideally rewrite the type definition to
```ts
declare function bodybuilder(): any;
export = bodybuilder; 
```
But I will heartily recommend more useful type definitions!